Added installation instructions to readme

JeremybellEU opened the pull request:
Added the installation instructions to the readme. This should explain new developers how to contribute to the project.
Please verify for both IDE's that this setup is correct and working.

JWGmeligMeyling commented on README.md:
```diff
@@ -4,3 +4,23 @@
+
+To succesfully run/test the application, a couple of steps should be followed to install various dependencies and configurations:
+
+1. After cloning, open the IDE of your preference (Eclipse/IntelliJ), import the project as maven project and make sure `target/metamodel` is a resource-folder of your project.
```
I should be a `source folder` in Eclipse, not `resource folder`.

Azkle commented:
Looks good to me.

JWGmeligMeyling commented on README.md:
```diff
@@ -4,3 +4,23 @@
+Click the `Environment` tab.
+    * (IntelliJ only) Go to `me.moodcat.core.App` and click on `edit configurations`.
+  * Click on `new`.
+As name set `database-password` and as value the database password.
```
You can also run the main from the test package and use the embedded H2 database instead.

JeremybellEU replied:
Does this use the correct persistence.properties? As it seems to me (after running) that it just runs the core with the inserted password.

JeremybellEU commented:
I rebased this branch onto master to get some more updates. Added some more clarification according to comments and also added a new exception to notify developers about misconfiguration.

GijsWeterings commented:
LGTM
